apitools/org.eclipse.pde.api.tools.ui/src/org/eclipse/pde/api/tools/ui/internal/wizards/ApiToolingSetupWizardPage.java

```diff
@@ -268,11 +269,12 @@ private IProject[] getInputProjects() {
 		return pjs.toArray(new IProject[pjs.size()]);
 	}
 
+	@SuppressWarnings("restriction")
 	private boolean acceptProject(IProject project) throws CoreException {
 		if (project == null) {
 			return false;
 		}
-		return (project.hasNature(JavaCore.NATURE_ID) && project.hasNature("org.eclipse.pde.PluginNature")) //$NON-NLS-1$
+		return project.hasNature(JavaCore.NATURE_ID) && PDE.hasPluginNature(project)
```

Comment:
I think we should have also a isJavaPluginNature(...) that checks for java+plugin nature).

Reply:
I assume you mean **has**JavaPluginNature...

Reply:
I had a look at this but didn't found anymore case where this would useful.
For now I just added a `PluginProject.isJavaProject()` method that can be used in independently of `PluginProject.isPluginProject()`.
If later more use cases are discovered for a combined method we can still add it.

Reply:
At laest one can create a Plugin Project without a java nature in the wizzard (even though it is seldom useful for me...)

Reply:
Yes, but I did the changes in this PR purely from a code point of view.